docs/getting-started/sdk/internal/index.md

```diff
@@ -38,6 +38,43 @@ To build the SDK, run the following command:
 cargo build
 ```
 
+### Web clients
+
+The SDK is integrated into the web clients as a WebAssembly module. To build the SDK for the web
+clients, run the following command:
+
+```bash
+cd crates/bitwarden-wasm-internal
+./build.sh
+```
+
+If you encounter any issues make sure that you have you have the required dependencies installed.
+See `crates/bitwarden-wasm-internal/README.md` for more information.
+
+### Mobile clients
+
+The SDK is integrated into the mobile clients as a shared library using the UniFFI framework. The
+SDK is built for each platform separately.
+
+#### Android
+
+The SDK is integrated into the Android client as a Maven dependency. The commands for building the
+Android library differ depending on which CPU architecture you are targeting. For more information,
+see the `crates/bitwarden-uniffi/kotlin/README.md` file.
+
+#### iOS
+
+The SDK is integrated into the iOS client as a Swift package. To build the SDK for iOS, run the
+following command:
+
+```bash
+cd crates/bitwarden-uniffi/swift
+./build.sh
+```
+
+If you encounter any issues make sure that you have you have the required dependencies installed.
+See `crates/bitwarden-uniffi/swift/README.md` for more information.
```

Comment:
Should we just ask that they follow the readmes under Android / iOS instead? I think we will want to iterate on these and try and converge the approaches slightly. And it would be nice to not. always have to go back and update these docs.

Reply:
Yeah I was trying to balance the effort of updating the docs with the convenience of having a guide with everything in one place. That's why I ended up not copying the dependency installation instructions, but just having a bunch of sections saying "read the README" didn't feel very useful 🤷‍♂️  Maybe I just rewrite this with focus on which crates are important and skip trying to add build instructions

Reply:
How about having something like

Web: `crates/bitwarden-wasm-internal`
iOS: `crates/bitwarden-uniffi/ios`
Android ...